safe/gis/raster/zonal_statistics.py

```diff
@@ -69,16 +87,28 @@ def zonal_stats(raster, vector, callback=None):
     result = analysis.calculateStatistics(None)
     LOGGER.debug(tr('Zonal stats on %s : %s' % (raster.source(), result)))
```

Comment:
Not your code, but may be you can just remove the `tr` while working on the file

Reply:
Travis is already happy and I need this commit. It doesn't break anything and this string is already translated ;-) Thanks